NEWS.md

```diff
@@ -15,9 +15,9 @@ for human readability.
   - multicomponent compressible Euler
   - acoustic perturbation equations
   - Lattice-Boltzmann equations
-- New structured mesh type `StructuredMesh`
 - Composable `FluxPlusDissipation` and `FluxLaxFriedrichs()`, `FluxHLL()` with adaptable
   wave speed estimates were added in [#493](https://github.com/trixi-framework/Trixi.jl/pull/493)
+- New curved structured mesh type `CurvedMesh`
```

Comment:
You are absolutely right about your comment. However, I decided to favor speed over thoroughness and thought that it might be easier to decide upon this once #500 is about to be merged. That way, we can have a discussion with all stakeholders and decide on the naming for both types of meshes at once. Further, by then Jesse's PR might have further matured, so we might be able to decide upon that as well.

Thus, if it is Ok with you, I'd just leave it as it is for now, with the understanding that it will have to be changed later.

And no, IMHO your comment is *not* bikeshedding: It is about naming things properly such that nobody gets confused and that using Trixi remains as intuitive as possible.

Reply:
Okay, then let's go ahead with the current name `CurvedMesh` for now as you proposed :+1: